web/src/components/FormAndReviewList.js

```diff
@@ -82,7 +88,7 @@ const FormAndReviewList = ({
             collapsedComponent={collapsed}
             expandedComponent={expanded}
             index={index}
-            initialCollapsed={item.initialCollapsed}
+            initialCollapsed={!(hasAdded && index === list.length - 1)}
```

Comment:
Debating whether to switch these up so it's `initialExpanded` instead, so the logic will be easier to think about. The `!` at the beginning here is easy to miss, and getting rid of it also causes a messy equation: `!hasAdded || index !== list.length - 1`

What do you think, @hbillings?

Reply:
It's not _awful_ to reason about, but I did have to read it a couple of times to make sure I got it. I'd be in favor of switching it to `initialExpanded`.

Reply:
Switched!